### DISCUSSION TITLE
Can create game, but nobody can join

### DISCUSSION THREAD

**Discussion by thogaertner** [Impostor/Impostor#399] [Q&A]
Hi,

I created the server, runs also successfully. I can create a game, but others cannot join it (connection time out . Other can also create a game on my server, but cannot join created game. Do you have any idea how to fix it?  Thank you! 

FYI: Using version 1.4.0 with multi roles mod v2.2.2.

**Comment by miniduikboot** [comment]
Hi there, you're likely running into some kind of firewall issue where your Impostor server is not properly port forwarded.
Your mod can also cause issues with server selection, please check their documentation if they say something about configuring custom servers
Please visit the Discord for more support, this forum isn't well monitored, so it will take longer to get a response
